File: Solutions/Easy/1925. Count Square Sum Triples/Solution.cs
public class Solution {
    public int CountTriples(int n) {
        int result = 0;
        for(int i = 5; i <= n; i++){
            for(int j = 4; j < i; j++){
                for(int k = 3; k < j; k++){
                    if(k*k + j*j == i*i){
                        result += 2;
                    }
                }
            }
        }
        return result;
    }
}

File: Solutions/Easy/1925. Count Square Sum Triples/Solution.py
class Solution:
    def countTriples(self, n: int) -> int:
        result: int = 0
        for i in range(5, n + 1):
            for j in range(4, i):
                for k in range(3, j):
                    result += 2 if k**2 + j**2 == i**2 else 0
        return result